dependencies.props

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+a1e3835a389e9953fba5a96988bf8b670fcae207<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Yeah, this "indicates" (in a loose sense) that a `DependencyBuildInfo` doesn't have a matching `[...]CurrentRef` element. Handling this better is tracked by https://github.com/dotnet/buildtools/issues/1374.

I think in this case, `ProjectNTfs-TestILC` doesn't have a matching `ProjectNTfs-TestILCCurrentRef` element. (And `ProjectNTfs-TestILC-PackageVersion` doesn't either, but I think that item should be removed entirely.)

You can test this out locally with `UpdateDependenciesAndSubmitPullRequest` with invalid GitHub credentials--it'll do these changes and then fail, so you can try out the fix.